modules/offers/client/views/offer-host-edit.client.view.html

```diff
@@ -124,6 +124,30 @@ <h4 id="maxGuests">How many guests can you accommodate?</h4>
                     </span>
                   </div>
                 </fieldset>
+
+                <br /><br />
+
+                <fieldset
+                  class="offer-showOnlyInMyCircles"
+                  ng-if="offerHostEdit.offer.status !== 'no'"
+                >
+                  <legend>
+                    <h4 id="showOnlyInMyCircles">
```

Comment:
No need ID here — other sections had ID because of `aria-labelledby="maxGuests"` needs.